src/domain/event/eventInfo/EventInfo.tsx

```diff
@@ -100,140 +101,112 @@ const EventInfo: React.FC<Props> = ({ event }) => {
     <div className={styles.eventInfo}>
       <div className={styles.contentWrapper}>
         {/* Date info */}
-        <div className={styles.infoWithIcon}>
-          <div className={styles.iconWrapper}>
-            <IconCalendarClock className={styles.icon} />
-          </div>
-          <div className={styles.iconTextWrapper}>
-            <h2 className={styles.title}>{t('event.info.labelDateAndTime')}</h2>
-
-            {!!startTime &&
-              getDateRangeStr({
+        <InfoWithIcon icon={<IconCalendarClock />}>
+          <h2 className={styles.title}>{t('event.info.labelDateAndTime')}</h2>
+
+          {!!startTime && (
+            <>
+              {getDateRangeStr({
                 start: startTime,
                 end: endTime,
                 locale,
                 includeTime: true,
                 timeAbbreviation: t('commons.timeAbbreviation'),
               })}
-            {startTime && (
               <button className={linkStyles.link} onClick={downloadIcsFile}>
                 {t('event.info.buttonAddToCalendar')}
                 <IconAngleRight />
               </button>
-            )}
-          </div>
-        </div>
+            </>
+          )}
+        </InfoWithIcon>
 
         {/* Other event times */}
         <OtherEventTimes event={event} />
 
         {/* Location info */}
-        <div className={styles.infoWithIcon}>
-          <div className={styles.iconWrapper}>
-            <IconLocation className={styles.icon} />
-          </div>
-          <div className={styles.iconTextWrapper}>
-            <h2 className={styles.title}>{t('event.info.labelLocation')}</h2>
-            <div className={styles.mobileOnly}>
-              {[locationName, streetAddress, district, addressLocality]
-                .filter((e) => e)
-                .join(', ')}
-            </div>
-            {locationName && (
-              <div className={styles.desktopOnly}>{locationName}</div>
-            )}
-            {streetAddress && (
-              <div className={styles.desktopOnly}>{streetAddress}</div>
-            )}
-            {district && <div className={styles.desktopOnly}>{district}</div>}
-            {addressLocality && (
-              <div className={styles.desktopOnly}>{addressLocality}</div>
-            )}
-            <Link isExternal={true} to={getServiceMapUrl(event, locale, false)}>
-              {t('event.info.openMap')}
-            </Link>
-          </div>
-        </div>
+        <InfoWithIcon icon={<IconLocation />}>
+          <h2 className={styles.title}>{t('event.info.labelLocation')}</h2>
+          <Visible below={'sm'}>
```

Comment:
Some of these use curly braces still, could be consistent with "sm"

Reply:
Fixed